content/docs/1.3.0/snapshots-and-backups/csi-snapshot-support/csi-volume-snapshot-associated-with-longhorn-backup.md

```diff
@@ -0,0 +1,143 @@
+---
+title: CSI VolumeSnapshot Associated with Longhorn Backup
+weight: 2
```

Comment:
Can we change this to weight 3?
I'd prefer to describe snapshot first then backup.

Reply:
Thank you. Fixed